en/_posts/2016-02-25-filebeat-logs.md

```diff
@@ -0,0 +1,267 @@
+  prospectors:
+  # Each - is a prospector. Below are the prospector specific configurations
+  # Paths that should be crawled and fetched. Glob based paths.
+  # To fetch all ".log" files from a specific level of subdirectories
+  # /var/log/*/*.log can be used.
+  # For each file found under this path, a harvester is started.
+  # Make sure not file is defined twice as this can lead to unexpected behaviour.
+    - 
+        paths:
+        - /var/log/apache2/access.log
+        input_type: log
+        document_type: apache
+        fields_under_root: true
+    -
+        paths:
+        - /var/log/apache2/error.log
+        input_type: log
+        document_type: apache-error
+        fields_under_root: true
+    -
+        paths:
+        - /var/log/syslog
+        input_type: log
+        document_type: syslog
+        fields_under_root: true
+    
+  # Name of the registry file. Per default it is put in the current working
+  # directory. In case the working directory is changed after when running
+  # filebeat again, indexing starts from the beginning again.
+  registry_file: /var/lib/filebeat/registry
+############################# Output ##########################################                                                                                                      
+# Configure what outputs to use when sending the data collected by the beat.
+# Multiple outputs may be used.
+output:
+  ### Logstash as output
+  logstash:
+    # The Logstash hosts
+    hosts: ["c002-my-paas-logs-hostname.in.laas.runabove.com:5044"]
+    worker: 1
+    tls:
+      # List of root certificates for HTTPS server verifications
+      certificate_authorities:
+      - /usr/local/etc/filebeat/laas-ca.crt
+############################# Logging #########################################
+# There are three options for the log ouput: syslog, file, stderr.
+# Under Windos systems, the log files are per default sent to the file output,
+# under all other system per default to syslog.
+logging:
+  # Send all logging output to syslog. On Windows default is false, otherwise
+  # default is true.
+  to_syslog: false
+  # Write all logging output to files. Beats automatically rotate files if rotateeverybytes
+  # limit is reached.
+  to_files: true
+  # To enable logging to files, to_files option has to be set to true
+  files:
+  # The directory where the log files will written to.
+    path: /var/log/
+    # The name of the files where the logs are written to.
+    name: filebeat.log
+    # Configure log file size limit. If limit is reached, log file will be
+    # automatically rotated
+    rotateeverybytes: 10485760 # = 10MB
+    # Number of rotated log files to keep. Oldest files will be deleted first.
+    keepfiles: 7
+  # Sets log level. The default log level is error.
+  # Available log levels are: critical, error, warning, info, debug
+level: info
+``` 
+
+####SSL CA Certificate 
+```bash
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you make it available for download somewhere? That would make automation even easier. Thanks.